pkg/cmd/cli/cmd/helper.go

```diff
@@ -23,23 +23,25 @@ func selectContainers(containers []kapi.Container, spec string) ([]*kapi.Contain
 }
 
 func handlePodUpdateError(out io.Writer, err error, resource string) {
```

Comment:
I know this is prexisting, but if we want something like this made pretty, why don't we have a generic translation is `CheckErr`?